marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/reader/HubInputPartition.java

```diff
@@ -16,15 +16,17 @@
     private Map<String, String> options;
     private JsonNode initializeResponse;
     private int partitionNumber;
+    private JsonNode endpointParams;
 
-    public HubInputPartition(Map<String, String> options, JsonNode initializeResponse, int partitionNumber) {
+    public HubInputPartition(JsonNode endpointParams, Map<String, String> options, JsonNode initializeResponse, int partitionNumber) {
```

Comment:
Let's make this consistent with the HubInputPartitionReader constructor, where endpointParams is the last argument out of 4.